index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>MERCY MUGAMBI</title>
+
+    <!--stylesheet-->
+    <link rel="stylesheet" href="/style.css" />
+  </head>
+  <body>
+    <header class="header-nav">
+      <div class="logo">MER<span>cy</span></div>
+
+      <nav class="hamburger">
+        <img src="/assets/images/bars.png" />
+      </nav>
+    </header>
+
+    <main class="heading">
+      <div class="header">
+        <h1>I’m MERCY <br />Glad to see <br />you!</h1>
+      </div>
+
+      <div class="heading-paragraph">
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+      </div>
+      <div class="heading-social">
+        <div class="icons-header">
+          <p>LET’S CONNECT</p>
+        </div>
+        <div class="icons">
+          <li>
+            <a href="https://twitter.com/MercyMugambi15" target="_blank"
+              ><img src="./assets/icons/twitter.png"
+            /></a>
+          </li>
```

Comment:
- [ ] I think would be semantically correct to wrap `<li>` tags under `<ul>` tag instead of a `<div>` tag